client/public/src/components/storePickup.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {PickupFlex, Flexrow, Flexcol} from '../styling/styles.jsx';
+import {StockLeft, EditBtn, PickupBtnDiv, Pickuptext, StoreLocationStyle} from '../styling/storeStyle.jsx';
+
+const StorePickup = ({qty, store}) => {
+
+  const capitalize = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  }
+
+  return (
+    <Flexrow>
+      <PickupFlex>
+        <Flexcol>
+          <div>
+            <Pickuptext qty={qty}>{qty > 0 ? 'Pick up today' : 'Out of stock'} </Pickuptext>
+            at&nbsp;
```

Comment:
in jsx, it's more elegant / easier to read to use {' '} to create empty content. Try {' '} and see if it works for you.

Reply:
Didn't think of that! That looks a lot more legible.